libbeat/docs/shared-docker.asciidoc

```diff
@@ -150,7 +150,7 @@ For example:
 ["source", "sh", subs="attributes"]
 --------------------------------------------
 docker run \
-  --mount type=source=$(pwd}/data,destination=/usr/share/{beatname_lc}/data
+  --mount type=source=$(pwd)/data,destination=/usr/share/{beatname_lc}/data
   --read-only
   {dockerimage}
```

Comment:
I managed to get it working with `type=bind`, I also noticed that some `\` are missing making copy/paste not to work, here is my fixed version:
```
docker run --rm \
  --mount type=bind,source=$(pwd)/data,destination=/usr/share/filebeat/data \
  --read-only \
  docker.elastic.co/beats/filebeat:8.15.2
```

Reply:
Awesome. Thanks @belimawr!

Here's how it looks now (I'm building the docs off of `main` but once published this would show an 8.15.2 Docker image):

![screen](https://github.com/user-attachments/assets/42e91388-5a4b-41b8-bf32-0c3ed2a92a1c)